File: Projects/codechef/TREEROOT.java
package codechef;

import java.util.Scanner;

class TREEROOT {
    public static void main(String[] args) throws java.lang.Exception {
        Scanner stdin = new Scanner(System.in);
        int T = stdin.nextInt();
        while (T-- > 0) {
            int sumRoot = 0;
            int sumCldr = 0;
            int N = stdin.nextInt();
            while (N-- > 0) {
                sumRoot += stdin.nextInt();
                sumCldr += stdin.nextInt();
            }
            System.out.println(sumRoot - sumCldr);
        }
        stdin.close();
    }
}

File: Projects/codechef/TREEROOT.kt
package codechef

import java.util.*

object TREEROOT {
    @JvmStatic
    fun main(args: Array<String>) {
        val stdin = Scanner(System.`in`)
        var T = stdin.nextInt()
        while (T-- > 0) {
            var sumRoot = 0
            var sumCldr = 0
            var N = stdin.nextInt()
            while (N-- > 0) {
                sumRoot += stdin.nextInt()
                sumCldr += stdin.nextInt()
            }
            println(sumRoot - sumCldr)
        }
        stdin.close()
    }
}